Assets/CodeBase/Entities/player/Player.cs

```diff
@@ -118,18 +148,29 @@ void moveDown()
     void moveLeft()
     { 
         playerRigidBody.AddForce(Vector2.left * moveSpeed);
+
+        if (_isFacingRight)
+        {
+            _isFacingRight = false;
+            transform.localScale = new Vector3(-transform.localScale.x, transform.localScale.y, transform.localScale.z);
```

Comment:
You can do GetComponent for SpriteRenderer and use that to change the flip boolean to accomplish this.